dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/TelemetryLongRunningTest.java

```diff
@@ -41,11 +41,9 @@ public class TelemetryLongRunningTest extends FATServletClient {
 
     public static final String SERVER_NAME = "Telemetry10LongRunning";
     public static final String APP_NAME = "LongRunningApp";
+    public static final int DEFAULT_TIMEOUT = 120 * 1000; // 120 seconds
```

Comment:
```suggestion
    public static final Duration DEFAULT_TIMEOUT = Duration.ofSeconds(120);
```

When storing a duration, usually we should use the `Duration` class, rather than an `int` or a `long` to make it easier to translate between units.

If we do want to store a duration as a number rather than a `Duration` object for some reason, it's important that we include the unit in the variable name so that we don't mix up units later. E.g. `DEFAULT_TIMEOUT_MILLIS`.